Jenkinsfile

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env groovy
+
+/**
+ * Build Contiki Applications
+ *
+ */
+
+
```

Comment:
Add buildDiscarder to delete old builds

    properties(
        [
            buildDiscarder(logRotator(numToKeepStr: '5')),
        ]
    )